.changeset/sweet-carpets-kiss.md

```diff
@@ -0,0 +1,35 @@
+---
+'braid-design-system': minor
+---
+
+---
+updated:
+  - useToast
+---
+
+**useToast:** Add neutral tone support
+
+Add support for `neutral` tone. When using a `neutral` tone, an icon may optionally be provided. For consistency, the tone of the icon is set to **secondary** and cannot be customised.
+
+**EXAMPLE USAGE:**
+```jsx
+import { useToast } from "braid-design-system"
+
+export default () => {
```

Comment:
Super picky, but could we not demo components using anonymous default exports? They break so much tooling.